packages/components/spacings/inline/README.md

```diff
@@ -1,10 +1,12 @@
 # Inline
 
+#### Description
+
 If you want to learn more about the spacing components take a look at
 [this](https://medium.com/eightshapes-llc/space-in-design-systems-188bcbae0d62)
 article.
 
-## Example
+#### Usage
 
 ```jsx
 import { Spacings } from '@commercetools-frontend/ui-kit';
```

Comment:
The import should be `import SpacingsInline from '@commercetools-uikit/spacings-inline';`

Same for the other ones.

PS: we can also mention to use the `spacings` preset

Reply:
oh no.